src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -410,89 +409,6 @@
     <Message Importance="High" Text="See '$(RepoConsoleLogFile)' for more information." Condition="Exists('$(RepoConsoleLogFile)') and '$(MinimalConsoleLogOutput)' == 'true'" />
   </Target>
 
-  <!--
-    Remove this target after arcade changes for publishing have flown to installer repo.
-    https://github.com/dotnet/source-build/issues/4207
-  -->
-  <Target Name="CopyRepoArtifacts"
-          DependsOnTargets="RepoBuild"
-          Condition="'$(IsUtilityProject)' != 'true' and Exists('$(RepoManifestFile)')"
-          Inputs="$(RepoManifestFile)"
-          Outputs="$(BaseIntermediateOutputPath)CopyRepoArtifacts.complete">
-    <Error Text="Repo manifest file doesn't exist." Condition="!Exists('$(RepoManifestFile)')" />
-
-    <XmlPeek XmlInputPath="$(RepoManifestFile)"
-             Query="Build/Artifact/@Path">
-        <Output TaskParameter="Result" ItemName="RepoManifestArtifact" />
-    </XmlPeek>
-
-   <!-- Error if a repository doesn't produce any shipping or non-shipping artifacts. -->
-    <ItemGroup Condition="'@(RepoManifestArtifact)' == ''">
-      <_ArtifactsNotCreatedReason Include="^ There may have been a silent failure in the repo build." />
-      <_ArtifactsNotCreatedReason Include="^ To confirm, check the build log file for undetected errors that may have prevented package creation: $(RepoConsoleLogFile)" Condition="'$(MinimalConsoleLogOutput)' == 'true'" />
-      <_ArtifactsNotCreatedReason Include="^ The packages may have been written to an unexpected directory. For example, some repos used bin/ and changed to artifacts/ to match Arcade. Check RepoManifestFile in $(MSBuildProjectFullPath) (currently '$(RepoManifestFile)')" />
-    </ItemGroup>
-
-    <Error Text="$(RepositoryName) produced no new source-built package identities. Known possible causes:%0A@(_ArtifactsNotCreatedReason, '%0A')"
-           Condition="'@(RepoManifestArtifact)' == ''" />
-
-    <!-- The repo symbol archive shouldn't be copied. -->
-    <ItemGroup>
-      <RepoSymbolsArchive Include="@(RepoManifestArtifact)"
-                          Condition="$([System.String]::Copy('%(Identity)').EndsWith('Symbols.$(RepositoryName)$(ArchiveExtension)'))" />
-      <RepoManifestArtifact Remove="@(RepoSymbolsArchive)" />
-    </ItemGroup>
-
-    <!-- Log the new repo artifacts -->
-    <Message Importance="High" Text="New artifact(s) after building $(RepositoryName):" />
-    <Message Importance="High" Text="  -> %(RepoManifestArtifact.Filename)%(RepoManifestArtifact.Extension)" />
```

Comment:
These messages were really useful so I wouldn't want to just delete them without having a replacement. I will add a target that parses the build manifest file and brings this back.

Reply:
Resolved with latest commit